explorer/src/context/main.tsx

```diff
@@ -1,17 +1,33 @@
+import { PaletteMode } from '@mui/material';
 import * as React from 'react';
+import { GatewayResponse, MixNodeResponse, ValidatorsResponse } from 'src/typeDefs/node-status-api-client';
```

Comment:
Might be best to name that file `src/typeDefs/explorer-api`?

When I started this I was using another API before the Explorer API existed. Apologies for creating naming confusion in the vue-js project!

Reply:
Ah right. Yep, i'll rename then to something more appropriate. Cheers.